activity.py

```diff
@@ -105,3 +83,10 @@ def sound_control(self, button):
         else:
             button.set_icon_name('speaker-muted-100')
             button.set_tooltip(_('Sound'))
+
+
+if __name__ == "__main__":
+    Gtk.init([])
+    activity = Activity(None)
+    activity.show_all()
+    Gtk.main()
```

Comment:
Why this change?

Reply:
`if __name__ == "__main__":` will ensures that they are executed only when the script is run directly

Reply:
Sorry, I thought this is the main file that's why I have included it.
I will remove this

Reply:
It looks like this change was made to test outside Sugar.  Have you tested inside Sugar?